virttest/libvirt_xml/devices/serial.py

```diff
@@ -23,11 +24,19 @@ def __init__(self, type_name='pty', virsh_instance=base.virsh):
                                tag_name='target', attribute='type')
         accessors.XMLAttribute('target_model', self, parent_xpath='/target',
                                tag_name='model', attribute='name')
-        accessors.XMLElementDict('alias', self, parent_xpath='/',
-                                 tag_name='alias')
         accessors.XMLElementList('sources', self, parent_xpath='/',
                                  marshal_from=self.marshal_from_sources,
                                  marshal_to=self.marshal_to_sources)
+        accessors.XMLElementDict('address', self, parent_xpath='/',
+                                 tag_name='address')
+        accessors.XMLAttribute('address_type', self, parent_xpath='/',
```

Comment:
I assume those attributes configuration are not needed, just accessors.XMLElementDict is enough

Reply:
Changed to use XMElementNest with Address class instead of XMLElementDict.

Reply:
Changed to use ElementDict only for no special needed for sub attribute.